pegasus/sites.v3/code.org/public/teach-draft.haml

```diff
@@ -0,0 +1,183 @@
+        .text-wrapper
+          %h3.heading-sm
+            =hoc_s(:teach_page_why_teach_cs_block_heading_03)
+          %p=hoc_s(:teach_page_why_teach_cs_block_desc_03)
+      %div.flex-container
+        %figure
+          %img{src: "/images/teach-page-why-cs-04.png", alt: ""}
+        .text-wrapper
+          %h3.heading-sm
+            =hoc_s(:teach_page_why_teach_cs_block_heading_04)
+          %p=hoc_s(:teach_page_why_teach_cs_block_desc_04)
+          %a{href: "/ai"}
+            =hoc_s(:call_to_action_explore_ai_resources)
+    %hr.dark
+    %div.centered
+      %h4=hoc_s(:teach_page_why_teach_cs_heading_02)
+      %a.link-button{href: CDO.studio_url("/catalog")}
+        =hoc_s(:call_to_action_explore_curriculum_catalog)
+
+%section.quote.bg-neutral-dark
+  .wrapper
+    %div.quotes-wrapper
+      %p.quote-text
+        =hoc_s(:teach_page_quote)
+      %p.no-margin-bottom Kevin Barry <span>•</span> CS Discoveries Teacher
+
+%section.free-curriculum
+  .wrapper
+    %h2.centered
+      =hoc_s(:teach_page_free_curriculum_heading)
+    %p.centered
+      =hoc_s(:teach_page_free_curriculum_desc)
+    %div.flex-container.justify-space-between.wrap-reverse
+      %ul.icon-bullet-list.full-width.col-60
+        %li
+          %div
+            %i{class: "#{icon_party_horn}"}
+          .text-wrapper
+            %h3=hoc_s(:teach_page_free_curriculum_list_heading_01)
+            %p.no-margin-bottom
+              =hoc_s(:teach_page_free_curriculum_list_desc_01)
+        %li
+          %div
+            %i{class: "#{icon_lightbulb_on}"}
+          .text-wrapper
+            %h3=hoc_s(:teach_page_free_curriculum_list_heading_02)
+            %p.no-margin-bottom
+              =hoc_s(:teach_page_free_curriculum_list_desc_02)
+        %li.no-margin-bottom
+          %div
+            %i{class: "#{icon_link}"}
+          .text-wrapper
+            %h3=hoc_s(:teach_page_free_curriculum_list_heading_03)
+            %p.no-margin-bottom
+              =hoc_s(:teach_page_free_curriculum_list_desc_03)
+      %figure.rounded-corners.col-33{style: "float: right"}
+    .clear
+    %hr.dark
+    %div.centered
+      %h4=hoc_s(:teach_page_free_curriculum_heading_02)
+      %div.flex-container.justify-center.wrap{style: "gap: 1rem"}
+        %a.link-button{href: CDO.studio_url("/catalog?grade=grade_1&grade=grade_2&grade=grade_3&grade=grade_4&grade=grade_5&grade=kindergarten")}
+          =hoc_s(:grade_level_label_elementary)
+        %a.link-button{href: CDO.studio_url("/catalog?grade=grade_6&grade=grade_7&grade=grade_8")}
+          =hoc_s(:grade_level_label_middle)
+        %a.link-button{href: CDO.studio_url("/catalog?grade=grade_10&grade=grade_11&grade=grade_12&grade=grade_9")}
+          =hoc_s(:grade_level_label_high)
+
+%section.resources.bg-neutral-light.centered
+  .wrapper
+    %h2=hoc_s(:teach_page_resources_heading)
+    %p=hoc_s(:teach_page_resources_desc)
+    -# RESOURCES TABS HERE
```

Comment:
The `h2` and `p` tag here will be replaced once [this PR](https://github.com/code-dot-org/code-dot-org/pull/52393) is merged.